### PR TITLE
Expose the totalCount and sumOfDurations query stats.

### PR DESCRIPTION
**Pull request by andreidan**
## Checklist

 - [ ] User relevant changes are recorded in ``CHANGES.txt``
 - [ ] Touched code is covered by tests
 - [ ] Documentation has been updated if necessary
 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -46,6 +46,9 @@ None
 Changes
 =======
 
+- Expose the total count and sum of durations metrics under the ``QueryStats``
+  MBean for ``QUERY``, ``INSERT``, ``UPDATE`` and ``DELETE`` statement types.
```
Comment (mfussenegger) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: Didn't we conclude to also expose COPY, DDL, MANAGEMENT and UNDEFINED?

**Comment by andreidan** [comment]
@mfussenegger pushed a fixup

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 2]
```diff
@@ -47,7 +47,8 @@ Changes
 =======
 
 - Expose the total count and sum of durations metrics under the ``QueryStats``
```
Comment (mfussenegger) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: Maybe link to the doc section? As advocated here https://github.com/crate/crate/pull/8232/commits/ce5ffa09034f7039b3b834a657eb21ea1eb97340#diff-8918b36d12905ff85de79bcbdcf052aaR32

File: blackbox/docs/admin/monitoring.rst [hunk 3]
```diff
@@ -127,13 +127,21 @@ Statements total count:
  - ``InsertQueryTotalCount``
  - ``UpdateQueryTotalCount``
  - ``DeleteQueryTotalCount``
+ - ``ManagementQueryTotalCount``
+ - ``DDLQueryTotalCount``
+ - ``CopyQueryTotalCount``
+ - ``UndefinedQueryTotalCount``
 
 Statements sum of durations:
```
Comment (mfussenegger) [inline blackbox/docs/admin/monitoring.rst]: I think the unit of the duration should be documented

File: enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java [hunk 4]
```diff
@@ -61,26 +64,67 @@ private Metrics createMetric(Classification classification, long duration) {
...
 
-        assertThat(metricsByCommand.get(StatementType.UNDEFINED).avgDurationInMs(), is(1.0));
-        assertThat(metricsByCommand.get(StatementType.UNDEFINED).statementsPerSec(), is(0.5));
+        assertThat(metricsByCommand.get(StatementType.DDL).totalCount(), is(1L));
+        assertThat(metricsByCommand.get(StatementType.DDL).sumOfDurations(), is(1L));
+        assertThat(metricsByCommand.get(StatementType.DDL).avgDurationInMs(), is(1.0));
+        assertThat(metricsByCommand.get(StatementType.DDL).statementsPerSec(), is(0.5));
 
+        assertThat(metricsByCommand.get(StatementType.ALL).totalCount(), is(7L));
+        assertThat(metricsByCommand.get(StatementType.ALL).sumOfDurations(), is(125L));
```
Comment (seut) [inline enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java]: afaik, the ALL type totalCount + sumOfDurations in never exposed, so we won't have to test it

File: enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java [hunk 5]
```diff
@@ -100,21 +144,33 @@ public void testCreateMetricsMapWithPreviouslyReadMetrics() {
...
 
-        assertThat(metricsByCommand.get(StatementType.UNDEFINED).avgDurationInMs(), is(1.0));
-        assertThat(metricsByCommand.get(StatementType.UNDEFINED).statementsPerSec(), is(0.5));
+        assertThat(metricsByCommand.get(StatementType.DDL).totalCount(), is(1L));
+        assertThat(metricsByCommand.get(StatementType.DDL).sumOfDurations(), is(1L));
+        assertThat(metricsByCommand.get(StatementType.DDL).avgDurationInMs(), is(1.0));
+        assertThat(metricsByCommand.get(StatementType.DDL).statementsPerSec(), is(0.5));
 
+        assertThat(metricsByCommand.get(StatementType.ALL).totalCount(), is(4L));
+        assertThat(metricsByCommand.get(StatementType.ALL).sumOfDurations(), is(65L));
```
Comment (seut) [inline enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java]: same here, ALL is not exposed and thus must not be tested

File: enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java (hunk 4 above)
Comment (andreidan) [reply enterprise/jmx-monitoring/src/test/java/io/crate/beans/QueryStatsTest.java]: but we're not testing what the bean is exposing here, but what internal map is containing, so I don't particularly think it hurts to test it (especially as it affects the size of the map)

**Comment by andreidan** [comment]
@mfussenegger @seut pushed a fixup

**Pull request merged**